src/core/store/modules/lines/lines.reducer.ts

```diff
@@ -231,6 +231,7 @@ const linesReducer = createReducer(linesInitialState, (builder) => {
       state.statusMap.user.getBorrowerPositions = { loading: true };
     })
     .addCase(getBorrowerPositions.fulfilled, (state, { meta, payload: { borrowerPositions } }) => {
+      console.log('positions in reducer', borrowerPositions);
       if (!borrowerPositions) return;
       const borrowerPositionsMap = borrowerPositions.reduce((obj, a) => ({ ...obj, [a.id]: a }), {});
       state.user.borrowerPositions = { ...state.user.borrowerPositions, ...borrowerPositionsMap };
```

Comment:
I thought i made this comment somewhere already but can we:
1. remove the `user.borrowerPositions` field on state.
2. change user.positions type to UserPositionSummary
3. setup selectors fo borrow/lend/arb positions using filters over user.positions